src/site/stages/build/process-cms-exports/transformers/node-page.js

```diff
@@ -46,6 +50,7 @@ function pageTransform(entity) {
 
   delete transformed.moderationState;
   delete transformed.metatag;
+  delete transformed.path;
```

Comment:
Yeah, this is probably worth a refactor after all. Like you said the other day.

That can be done in a different PR.